backend/controllers/user/users.js

```diff
@@ -329,15 +333,16 @@ module.exports = {
     );
   },
   toggleFollow: async (req, res) => {
-    const { follower_id, following_id } = req.body;
+    const { currentUserId } = req;
+    const { following_id } = req.body;
 
-    if (!follower_id || !following_id) {
+    if (!following_id) {
       return res.send(
-        ResponseHelper.generateResponse(400, "Missing follower or following id")
+        ResponseHelper.generateResponse(400, "Missing following id")
```

Comment:
- `400` were used for bad requests. I prefer to use in the scenario `404` or `403`, please do your own good judgement base in this https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
Hello sir, I think the error message does not fall under `404` nor `403`. The error message is about a missing parameter when the request is sent by the client.

![image](https://user-images.githubusercontent.com/111732984/198209484-c0682cd7-39cc-4b98-b7ef-07b49c5b5d9d.png)

Reply:
sorry i thought you query the parameter. Yeah i'm okay with `400`.